docs/getting-started.md

```diff
@@ -2,7 +2,101 @@
 
 This guide is here to help you get started with this repository, primarily if you've been contributing to the previous version of ownCloud's documentation, which used reStructuredText and Sphinx-Doc.
 
-## Initial Steps
+## For Non-Developers
+
+Even if you’re not a developer, you can contribute to the ownCloud documentation by [creating new issues](#creating-new-issues) and contributing to the discussion around them.
+You don’t need to be familiar with how to write or edit code, nor install any software.
+All that you need is a web browser, such as [Google Chrome](https://www.google.com/chrome/index.html), [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/), [Apple Safari](https://www.apple.com/safari/), or [Microsoft Edge](https://www.microsoft.com/en-us/windows/microsoft-edge).
+
+### Initial Steps
+
+Before you can contribute to the documentation, you need to:
+
+1. [Have a GitHub account](#have-a-github-account).
+2. [Know the AsciiDoc basics.](#asciidoc-basics)
+
+<a name="have-a-github-account"></a>
+#### Have a GitHub Account
+
+If you don't already have an account, you can [sign up for a new one](https://github.com/).
+You only need to provide a _username_, _email address_, and a _password_.
+
+![Create a new GitHub account.](images/getting-started/non-developers/github-signup.png)
+
+<a name="asciidoc-basics"></a>
+#### Know the AsciiDoc Basics
+
+While [the AsciiDoc file format](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/) is quite rich, you're likely to only need a small portion of it, a lot of the time.
+In the example below, you can see a small example which shows how to format _headers_, _paragraphs_, _links_, _lists_, and _source code_.
+
+```asciidoc
+= Top-Level Header
+
+An introduction to http://asciidoc.org[AsciiDoc].
+
+== Second-Level Header
+
+* item 1
+* item 2
+
+[source,ruby]
+====
+puts "Hello, World!"
+====
+```
+
+Have a read through [the AsciiDoctor Syntax Quick Reference](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/) to familiarise yourself with the format's essentials.
```

Comment:
You may want to add:
`Please also refer to [Best Practices and Tips](./docs/best-practices.md) for more information.`